web/src/components/Collapsible.tsx

```diff
@@ -18,6 +18,7 @@ const CollapsibleHeader = styled.div`
   display: flex;
   justify-content: space-between;
   cursor: pointer;
+  outline: none;
```

Comment:
rm ugly outline when clicking on collapsible header

Reply:
When was this shown? Can't reproduce or maybe I am just blind :eyes:

Reply:
click on the collapseHeader and open content / close